File: shared/src/commonMain/kotlin/pe/ecolecta/domain/model/EstadoSeguimiento.kt
package pe.ecolecta.domain.model

enum class EstadoSeguimiento {
    INACTIVO,
    /** El seguimiento ya arrancó pero todavía no llegó ninguna coordenada real (ni la inicial ni tras recuperar señal). */
    BUSCANDO,
    ACTIVO,
    SIN_CONEXION,
    SIN_SENAL,
    PERMISO_DENEGADO,
    ERROR_ALMACENAMIENTO,
    ERROR_CAPTURA,
    NO_DISPONIBLE_PLATAFORMA,
}

File: web/app/Domain/Seguimiento/EstadoSeguimiento.php
<?php

namespace App\Domain\Seguimiento;

enum EstadoSeguimiento: string
{
    case Inactivo = 'inactivo';
    case Buscando = 'buscando';
    case Activo = 'activo';
    case PermisoDenegado = 'permiso_denegado';
    case NoDisponible = 'no_disponible';
}
